homeassistant/components/notify/webostv.py

```diff
@@ -21,12 +22,13 @@
 WEBOSTV_CONFIG_FILE = 'webostv.conf'
 HOME_ASSISTANT_ICON_PATH = os.path.join(os.path.dirname(__file__), '..',
```

Comment:
This file is no longer available in this repo.

Reply:
Should I remove this option or set default to None and don't send icon

Reply:
Yes, let's not send icon.